phys/module_sf_noahdrv.F

```diff
@@ -785,6 +789,11 @@ SUBROUTINE lsm(DZ8W,QV3D,P8W3D,T3D,TSK,                      &
 ! use mid-day albedo to determine net downward solar (no solar zenith angle correction)
         SOLNET=SOLDN*(1.-ALBEDO(I,J))
         PRCP=RAINBL(i,j)/DT
+        IF(IRR_CHAN(i,j).NE.0) THEN
```

Comment:
May need a PRESENT check for IRR_CHAN here and in lsm_mosaic.
NMM will run Noah too, so any irrigation things have to be blocked for it.

Reply:
corrected now